From b76b0fb4697c0fdceddde11f1d462b25c972400b Mon Sep 17 00:00:00 2001
From: Dan Poltawski <dan@moodle.com>
Date: Sat, 30 Jul 2016 19:35:21 +0100
Subject: [PATCH 1/1] Too far in future date.

Hopefully in 700 years we won't still be using these tests
---
 cache/stores/static/version.php | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cache/stores/static/version.php b/cache/stores/static/version.php
index 925c395..e70673b 100644
--- a/cache/stores/static/version.php
+++ b/cache/stores/static/version.php
@@ -27,6 +27,6 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version   = 2016052300;    // The current module version (Date: YYYYMMDDXX)
+$plugin->version   = 2716052300;    // The current module version (Date: YYYYMMDDXX)
 $plugin->requires  = 2016051900;    // Requires this Moodle version.
-$plugin->component = 'cachestore_static';  // Full name of the plugin.
\ No newline at end of file
+$plugin->component = 'cachestore_static';  // Full name of the plugin.
-- 
2.9.0

